kb/sd_of_ann_actions/concepts/concept_all_layer_neurons_weighted_sum_calculating_action.scs

```diff
@@ -0,0 +1,47 @@
+concept_all_layer_neurons_weighted_sum_calculating_action
```

Comment:
Rename to action_calculate_weighted_sum_of_all_neurons_of_layer like in single_layer_perceptron_scp.scs